tensorflow/python/ops/special_math_ops.py

```diff
@@ -235,14 +234,14 @@ def einsum(equation, *inputs, **kwargs):
 
       output_axis_labels = ''.join(
           sorted(ax for ax in indices if counts[ax] == 1))
-
     for a in axis_labels:
       for input_labels in input_axis_labels:
+        if len(input_axis_labels) == 1 and input_labels.count(a) == 2 and input_labels==input_labels[::-1]:
```

Comment:
Can we update the doc?

Reply:
Updated the doc, let me know what you think